src/ui/src/core_components/other/useJSONParser.ts

```diff
@@ -0,0 +1,23 @@
+import { ComputedRef, ref, watch } from "vue";
+import { JSONData } from "../../wds/WdsJsonViewer.utils";
+
+export function useJSONParser(rawJSON: ComputedRef<string>) {
```

Comment:
The composables are on `/renderer`, please see if you can refactor for it to be there.

If it's generic and can potentially serve other components, it can go on `/renderer`.

Reply:
This one become obsolete since I can use `FieldType.Object` :+1: